Source/WebCore/page/Quirks.cpp

```diff
@@ -1596,4 +1596,14 @@ bool Quirks::shouldAdvertiseSupportForHLSSubtitleTypes() const
 }
 #endif
 
+// apple-console.lrn.com (rdar://106779034)
+bool Quirks::shouldDisablePopoverAttributeQuirk() const
+{
+    if (!needsQuirks())
+        return false;
+
+    auto host = m_document->topDocument().url().host();
+    return equalLettersIgnoringASCIICase(host, "apple-console.lrn.com"_s);
```

Comment:
do we have any reason to believe it's only `apple-console.lrn.com` rather than all subdomains of `lrn.com`?

Reply:
I'm intentionally targeting `apple-console.lrn.com` for now